### ISSUE TITLE
Improvements to theatre mode

### ISSUE THREAD

**Issue by peepopoggers** [FreeTubeApp/FreeTube#1315]
These are some miscellaneous changes that would improve usability.

Improvement #1 - The theatre icon is not very recognizable.
![image](https://user-images.githubusercontent.com/72892531/119242511-48545500-bb56-11eb-876c-2f36cc2c1303.png)
It is indifferent from fullscreen, and tv-cast or something similar. A better option would be for it to be intergrated into the video player but I am guessing its not possible for now, so an icon with arrows expanding and one with arrows inwards would signify the purpose much better. Or IMO it could just be a wide box, rather than monitor icon.

Scuffed example:
![image](https://user-images.githubusercontent.com/72892531/119242607-37581380-bb57-11eb-83e8-f98f14992e69.png)

Improvement #2 - Theatre mode blocks switch to normal view without scroll.
![image](https://user-images.githubusercontent.com/72892531/119242680-dda41900-bb57-11eb-9aa5-e84a1cca73a3.png)
When theatre mode is enabled, the useful buttons are moved out of the way, which makes it annoying to take actions. Theatre size could be lessened a little bit to make these buttons appear.

Example (width 1300px):
![image](https://user-images.githubusercontent.com/72892531/119242670-c107e100-bb57-11eb-898c-7de8c0223fdb.png)

It could be dynamically sized, up to a certain point, to always keep these main buttons shown, while also making video player as wide as possible.

**Comment by Svallinn** [comment]
If you have an issue with the TV icon, read [this thread](https://github.com/FreeTubeApp/FreeTube/issues/1180) and, after doing so, you can take the discussion up to those individuals and convince them otherwise
TL;DR - We are using what *FontAwesome* can provide us with and a few individuals decided that icon was the best option

**Comment by peepopoggers** [comment]
@Svallinn 

Alright, will do.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I can see the icons while theater mode is enabled. 

In my opinion it would not be a good idea to sacrifice valuable screen width for some icons.

**Comment by peepopoggers** [comment]
I mean, it could just show enough to show the buttons

**Comment by peepopoggers** [comment]
Can you send me a screenshot of how it looks?

**Comment by peepopoggers** [comment]
I think dynamically sized would be good. lets say - it would become smaller with height until it reaches a minimum size - then details below would go away until the video player is reached, where it would resize again.
I use different vertical margins for programs, so a dynamic approach like this would make sense.

**Comment by GilgusMaximus** [comment]
First issue already discussed before. We will add it to the player eventually.

Second issue, I will not add anything. Imo it is fine like it is, but I also have no problem pushing things together if the video size is still reasonable

**Comment by peepopoggers** [comment]
ok, for #3, my bad.

I think for theatre buttons, they could be partially clipped, so they are partially visible so they can still be clicked/in sight and also the theatre mode will only be a little smaller

**Comment by peepopoggers** [comment]
![image](https://user-images.githubusercontent.com/72892531/119367332-bd846f00-bca9-11eb-8e25-0158c64c04c9.png)

**Comment by BrainstormSurge** [comment]
I wouldn't mind a couple options that could be changed within the settings depending on your preference. For example, I like the Invidious UI as it's always in Theater Mode and any recommended videos can't be seen unless you scroll down. Of course you can hide recommended videos in FreeTube but you get what I mean. 

With Invidious there's only the video title and the video window takes up the rest of the screen pretty much. Video stats need to be scrolled down to see and are a lot more detailed then YouTube shows such as ratings and engagement. Obviously you wouldn't need to copy the UI of Invidious 1 to 1 but it's certainly something to get inspired by.

However you choose to improve Theater Mode I would like it as the other person stated where the video takes up as much screen real estate as possible without being full window. So maybe just the video title and that's it like Invidious but the rest of the UI however you feel it should be.

You could move the buttons such as Toggle Theater Mode in the left bar in the bottom-left stacked on each other or you could even move them to the right in the bottom-right. Could even do both and have theater-mode settings to customize how someone would like it, or maybe in the future so you can ship Better Theater Mode sooner.

Just some ideas.

**Comment by PrestonN** [comment]
I think I'm going to close this issue, and I'll explain why.

A couple of others have mentioned the icon situation, so I'll skip this.

As far as the video sizing goes, we are never going to find a solution that works for everyone. Everyone has different size screens and the videos themselves can be published with different aspect ratios. One setup for sizing might look best for you, but could end up looking odd for someone else and vice versa. Outside of some way to allow the user to manually set this sizing, which we likely won't be doing, then attempting to fine tweak it to fit what everyone wants isn't going to have a conclusion.

Because of this, I don't think that our current implementation for sizing is going to be changing any time soon.
